app/src/main/kotlin/com/simplemobiletools/dialer/extensions/Context.kt

```diff
@@ -44,3 +45,11 @@ fun Context.areMultipleSIMsAvailable(): Boolean {
         false
     }
 }
+
+private var toneGeneratorHelperInstance: ToneGeneratorHelper? = null
```

Comment:
there should be no variables in extensions, it becomes messy. Just store the helper in the activity.